src/deploy/table_upload.sql

```diff
@@ -8,13 +8,17 @@ CREATE TABLE maevsi.upload (
   id             UUID PRIMARY KEY DEFAULT gen_random_uuid(),
   account_id     UUID NOT NULL REFERENCES maevsi.account(id),
   size_byte      BIGINT NOT NULL CHECK (size_byte > 0),
-  storage_key    TEXT UNIQUE
+  storage_key    TEXT UNIQUE,
+  file_name      TEXT,
+  file_type      TEXT NOT NULL DEFAULT 'image'
```

Comment:
I'll drop the `file_` prefix, we can add it to all column names later (or rename the table to `file` which might even make more sense, but that'd be a breaking change).